README.md

```diff
@@ -3,13 +3,44 @@
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+Speed depends on many things besides the asymptotic analysis; it is based on hardware,  Other processes running besides the program that you are testing the performance of, and the secondary processes inside the program that are ignored in asymptotic analysis.
```

Comment:
When we analyze a program we analyze all of it, including any auxiliary functions.